src/test/script_standard_tests.cpp

```diff
@@ -128,6 +128,20 @@ BOOST_AUTO_TEST_CASE(script_standard_Solver_success)
     BOOST_CHECK(solutions[0] == std::vector<unsigned char>{16});
     BOOST_CHECK(solutions[1] == ToByteVector(uint256::ONE));
 
+    // TxoutType::ANCHOR
+    std::vector<unsigned char> anchor_bytes{0x4e, 0x73};
```

Comment:
nit:  Rather than repeating `0x4e, 0x73` in multiple places, maybe this could be macro'd out (or similar)?  At minimum, this might help increase clarity and reduce adding explanatory comments for the magic numbers.  Also seems more consistent with DRY principle, but I doubt these values will ever change (so maybe not a ton of value-added).  Alternatively, could be done in a follow-up PR.

Reply:
I'd be willing to entertain code changes in followups for this. Leaving as-is.